bot/cogs/watchchannels/talentpool.py

```diff
@@ -220,6 +206,36 @@ async def edit_reason_command(self, ctx: Context, nomination_id: int, *, reason:
 
         await ctx.send(f":white_check_mark: Updated the {field} of the nomination!")
 
+    @Cog.listener()
+    async def on_member_ban(self, guild: Guild, user: Union[User, Member]) -> None:
+        """Remove `user` from the talent pool after they are banned."""
+        await self.unwatch(user.id, "User was banned.")
+
+    async def unwatch(self, user_id: int, reason: str) -> bool:
+        """End the active nomination of a user with the given reason and return True on success."""
+        active_nomination = await self.bot.api_client.get(
+            self.api_endpoint,
+            params=ChainMap(
+                self.api_default_params,
+                {"user__id": str(user_id)}
```

Comment:
Reading this code again (I think I wrote it), I wonder why I put the defaults *before* the `user__id` dict in the `ChainMap`. The lookup order is from left to right, not right to left, meaning that the default values in `api_default_params` now take precedence over the specific values inserted as params.

It may not matter too much in this version, as I don't think our defaults will ever include a `user__id` field, but if we ever add something to the second dictionary hoping to "override" a default, it won't work. Not really in the scope of your PR, but my personal feeling is that I should have picked the reverse order in the original.

Reply:
Seems reasonable to me. I don't really see any reason not to swap these?